webpack/webpack.config.shared.js

```diff
@@ -10,22 +10,19 @@ module.exports = {
         filename: './[name]_bundle.js'
     },
     module: {
-        loaders: [
-            { test: /\.js$/, exclude: /node_modules/, loader: 'react-hot!babel' },
+        rules: [
+            { test: /\.js$/, exclude: /node_modules(?!(\/|\\)openfin-layouts)/, loader: 'babel-loader' },
             { test: /\.less$/, loader: 'style-loader!css-loader!less-loader' },
```

Comment:
It might be nicer to use `use: [ { loader: 'style-loader' }, { loader: 'css-loader' }, ... ]` here

Reply:
Given we don't apply any option to those loaders, don't you think we would have `loader` written all over the place for little benefit ?

Reply:
I think it's an aesthetic opinion. For me I prefer that syntax to chaining with `!`, but you may feel differently.